src/TMedia.h

```diff
@@ -121,6 +122,7 @@ private slots:
     void stopAllMediaPlayers();
     void setMediaPlayersMuted(const TMediaData::MediaProtocol mediaProtocol, const bool state);
     void transitionNonRelativeFile(TMediaData& mediaData);
+    QString getStreamUrl(TMediaData& mediaData);
```

Comment:
Is it intended to modify the reference passed? If not - it is probably good policy to enforce `const`-ness
```suggestion
    QString getStreamUrl(const TMediaData& mediaData);
```

Reply:
Done